src/V3Config.cpp

```diff
@@ -454,12 +454,12 @@ class V3ConfigResolver final {
     ~V3ConfigResolver() = default;
 
 public:
-    static V3ConfigResolver& s() {
+    static V3ConfigResolver& s() VL_MT_SAFE {
         static V3ConfigResolver s_singleton;
         return s_singleton;
     }
     V3ConfigModuleResolver& modules() { return m_modules; }
-    V3ConfigFileResolver& files() { return m_files; }
+    V3ConfigFileResolver& files() VL_MT_SAFE { return m_files; }
```

Comment:
While returning a handle is safe, there needs to be a lock to use it, so this seems to be dangerous.

Reply:
Right. I removed this change from this PR.